app/models.py

```diff
@@ -1044,6 +1046,8 @@ def check_code(self, cde):
     NOTIFICATION_TECHNICAL_FAILURE,
     NOTIFICATION_TEMPORARY_FAILURE,
     NOTIFICATION_PERMANENT_FAILURE,
+    NOTIFICATION_PENDING_VIRUS_CHECK,
+    NOTIFICATION_VIRUS_SCAN_FAILED,
```

Comment:
should `VIRUS_SCAN_FAILED` also be put in the `NOTIFICATION_STATUS_TYPES_FAILED` list on line 1009?

Reply:
👍 I've added it in to the list